core/commands/escrow/escrow.go

```diff
@@ -0,0 +1,151 @@
+package escrow
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/TRON-US/go-btfs/core/commands/storage"
+
+	config "github.com/TRON-US/go-btfs-config"
+	"github.com/gogo/protobuf/proto"
+	ic "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/tron-us/go-btfs-common/crypto"
+	escrowpb "github.com/tron-us/go-btfs-common/protos/escrow"
+	"github.com/tron-us/go-btfs-common/protos/ledger"
+	"google.golang.org/grpc"
+)
+
+func NewContract(configuration *config.Config, id string, payerID string, recverID string, price int64) *escrowpb.EscrowContract {
+	payerAddr := []byte(payerID)
+	recverAddr := []byte(recverID)
+	// TODO: Make escrow GuardAddress configurable
+	//authAddress := configuration.Services.GuardPubKeys[0]
```

Comment:
you should use this line, and make your local `~/.btfs/config` change the value to `guardAddress`

Reply:
ok, thanks for clarify that